ruby/vampires.rb

```diff
@@ -1,15 +1,58 @@
-puts "What is your name?"
-name = gets.chomp
-puts "How old are you?"
-age = gets.chomp
-puts "What year were you born?"
-year = gets.chomp
-puts "Our company serves garlic bread. Should we order some for you (y/n)"
-order = gets.chomp
-puts "Would you like to enroll in the company's health insurance? (y/n)"
-insurance = gets.chomp
-
-puts case
-	when name == ("Drake Cula" || "Tu Fang")
-		puts "Definitely a vampire."
-end 
+#Repeating Loop
+puts "How many employees will be processed?"
+repeat = gets.chomp.to_i
+repeat.times do
+
+	#Initial Questionnaire
+	puts
+	puts "What is your name?"
+	name = gets.chomp
+	puts "How old are you?"
+	age = gets.to_i
+	puts "What year were you born?"
+	year = gets.to_i
+	puts "Our company serves garlic bread. Should we order some for you? (y/n)"
+	order = gets.chomp.downcase
+	ordered_garlic = order == "y"
+	puts "Would you like to enroll in the company's health insurance? (y/n)"
+	insurance = gets.chomp.downcase
+	enroll_insurance = insurance == "y"
+
+	correct_age = (age == 2017 - year)

```

Comment:
I would like to see you follow the age convention in the instructions.  We want to know if you the employee is lying about their age.

Lets consider checking the current year against how old they say they are and what year they say that they were born in.  Investigate the Ruby `Time` class and see if you can come up with something creative and dynamic.